gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -339,6 +346,24 @@ gis_summary_page_constructed (GObject *object)
                             "lost forever when you restart or shutdown the computer.</b>"));
   }
 
+  /* Override the contents if needed */
+  summary_mode = gis_driver_conf_get_string (GIS_PAGE (page)->driver,
```

Comment:
Have you tested whether the text gets updated when you change translation on the first page? (I accept that the live session stuff also makes the same assumption that the page will be torn down and rebuilt if the locale changes.) Otherwise at least some of this should live in locale_changed below.

Reply:
I haven't. It's important that translations work, that's why I didn't want the text added to the conf file, but it's not super important if in Hack images that bit is not translated (as we're supporting only English for now).